### ISSUE TITLE
PORTS 127.0.0.1 displayed when running - docker container ls command on macOS 

### ISSUE THREAD

**Issue by Milos88kg** [aerokube/selenoid#967]
Hi! 
I am trying to implement selenoid in my test project. Have already created test.xml, browsers.json file 

Successfully executed docker commands for starting ggr, selenoid and selenoid ui.

- When execute in terminal: curl http://'my ip':4445/status i receive correct browsers.json validation: {"total":5,"used":0,"queued":0,"pending":0,"browsers":{"chrome":{"84.0":{}},"firefox":{"78.0":{},"79.0":{}},"opera":{"68.0":{},"69.0":{}}}}

- When execute in terminal: curl http://test:test-password@'my-ip':4444/wd/hub/status i receive: 
{"value":{"message":"Ggr 1.6.7 built at 2020-03-27_12:50:09PM","ready":true}}

I have successfully run some tests on http://test:test-password@'my-ip':4444/wd/hub

The problem is when i execute - docker container ls command, i receive PORTS for all 3 containers to be 127.0.0.1... instead of: my-ip:44444 / my-ip:4445 / my-ip:8080 ? 

Here are the screenshots from terminal and test.xml file

![Screen Shot 2020-08-10 at 19 57 15](https://user-images.githubusercontent.com/50324479/89814881-24360300-db44-11ea-844a-07eceb7e84e1.png)
![Screen Shot 2020-08-10 at 19 55 18](https://user-images.githubusercontent.com/50324479/89814906-2e580180-db44-11ea-8afb-82722bc692c6.png)

Thanks! Milos

**Comment by vania-pooh** [comment]
@Milos88kg I don't think this is a Selenoid \ Ggr issue. Could you please show the command your are using to start one of them?

**Comment by Milos88kg** [comment]
Selenoid: 
docker run -d                                   \
--name selenoid                                 \
-p 'my-ip':4445:4444                                    \ 
-v /var/run/docker.sock:/var/run/docker.sock    \
-v ~/selenoid/config/:/etc/selenoid/:ro \
aerokube/selenoid:latest-release

Ggr:
docker run -d --name ggr -v /private/etc/grid-router:/etc/grid-router:ro -p 'my-ip':4444:4444 aerokube/ggr:latest-release  

Selenoid-UI:
docker run -d --name selenoid-ui --restart always -p 'my-ip':8080:8080 aerokube/selenoid-ui --selenoid-uri=http://'my-ip':4445/

**Comment by vania-pooh** [comment]
@Milos88kg seems to be correct. I would check Docker documentation regarding this port binding issue. There is nothing special in Selenoid \ Ggr as they also can work as a binary.

**Comment by Milos88kg** [comment]
Okay, thanks for info
